lib/Backends/CMakeLists.txt

```diff
@@ -38,7 +38,8 @@ target_link_libraries(Backend
                         CodeGen
                         Graph
                         IR
-                        Optimizer)
+                        Optimizer
```

Comment:
Why would the backends, who's job is to generate code, depend on the quantizer? The quantizer is just a procedure that mutates the graph.

Reply:
Agreed, this was because the `optimizeFunction()` lives in the Backend class, which I don't think makes sense anyway. I put up a PR to move `optimizeFunction()` to Optimizer in https://github.com/pytorch/glow/pull/2703.